src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToCopyPPFileToOutput.cs

```diff
@@ -0,0 +1,99 @@
+
+using System.IO;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using System.Xml.Linq;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeHaveAPpContentFile : SdkTest
+    {
+        public GivenThatWeHaveAPpContentFile(ITestOutputHelper log) : base(log)
+        {}
+
+        [Fact]
+        public void It_copies_to_output_successfully()
+        {
+            var packageReference = GetPackageReference();
+
+            TestProject testProject = new TestProject()
+            {
+                Name = "CopyPPToOutputTest",
+                IsSdkProject = true, 
+                IsExe = true, 
+                TargetFrameworks = "netcoreapp3.0"
+            };
+            testProject.PackageReferences.Add(packageReference);
+            testProject.AdditionalProperties.Add("RestoreAdditionalProjectSources", Path.GetDirectoryName(packageReference.NupkgPath));
+            //  Use a test-specific packages folder
+            testProject.AdditionalProperties["RestorePackagesPath"] = @"$(MSBuildProjectDirectory)\..\pkg";
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            var restoreCommand = new RestoreCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name));
+            restoreCommand.Execute()
+                .Should()
+                .Pass();
+
+            var buildCommand = new BuildCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name));
+            buildCommand.Execute()
+                .Should()
+                .Pass();
+
+            var outputPath = Path.Combine(testAsset.TestRoot, testProject.Name, "bin", "Debug", "netcoreapp3.0");
+            File.Exists(Path.Combine(outputPath, packageReference.ID + ".dll")).Should().BeTrue();
+            File.Exists(Path.Combine(outputPath, "Nontransformed.ps1")).Should().BeTrue();
+            File.Exists(Path.Combine(outputPath, "Test.ps1")).Should().BeTrue();
+        }
+
+        private TestPackageReference GetPackageReference()
+        {
+            var referencedPackage = new TestProject()
+            {
+                Name = "CopyPPTilesToOutput",
+                TargetFrameworks = "netcoreapp3.0",
+                IsSdkProject = true
+            };
+
+            var packageAsset = _testAssetsManager.CreateTestProject(referencedPackage);
+            WriteFile(Path.Combine(packageAsset.TestRoot, referencedPackage.Name, "Nontransformed.ps1"), "Content file");
+            WriteFile(Path.Combine(packageAsset.TestRoot, referencedPackage.Name, "Test.ps1.pp"), "Content file");
+            packageAsset = packageAsset
+                .WithProjectChanges(project => AddContent(project));
+
+            var restoreCommand = new RestoreCommand(Log, packageAsset.TestRoot, referencedPackage.Name);
```

Comment:
Nick made a change to our tests where a separate `RestoreCommand` is usually not necessary.  The `PackCommand` (and most other commands) will restore implicitly, which makes the tests run faster and matches the developer experience (`dotnet pack` also implicitly restores).